src/types/index.ts

```diff
@@ -12,7 +12,7 @@ type Context = {
 type ExtensionMap = Map<string, Set<string>>;
 
 type Signature = {
-  Args: Map<string, string[]> | undefined;
+  Args: string[] | undefined;
```

Comment:
For simplicity, we will consider (for now) every argument's type to be `unknown`.